nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -156,6 +163,59 @@ private void processClassPath(String[] args) {
     return Sets.union(nonNullPositions, jiNonNullParams).immutableCopy();
   }
 
+  @Override
+  public NullnessHint onDataflowVisitMethodInvocation(
+      MethodInvocationNode node,
+      Types types,
+      AccessPathNullnessPropagation.SubNodeValues inputs,
+      AccessPathNullnessPropagation.Updates thenUpdates,
+      AccessPathNullnessPropagation.Updates elseUpdates,
+      AccessPathNullnessPropagation.Updates bothUpdates) {
+    if (!config.isJarInferUseReturnAnnotations()) return NullnessHint.UNKNOWN;
```

Comment:
Nit. I'd rather this and the ones below be in the form:

```
if (...) {
   ...;
}
```

It's easier to be aware of the control flow that way when glancing at the code.